### PR TITLE
add cleanup-BaseApp.sh

### PR DESCRIPTION
**Pull request by eszlari**
...and some cleanups in the manifest.

The script can be used in apps by adding:
```
cleanup-commands: /app/cleanup-BaseApp.sh
```

### CODE REVIEW

File: io.qt.qtwebengine.BaseApp.json [hunk 1]
```diff
@@ -97,15 +85,38 @@
...
+                    "dest-filename": "cleanup-BaseApp.sh",
+                    "//": "https://github.com/flatpak/flatpak-builder/issues/233",
+                    "commands": [
+                        "export ARCH_TRIPLE=$(gcc --print-multiarch)",
+                        "rm -rfv /app/bin/pprof",
+                        "rm -rfv /app/include",
+                        "rm -rfv /app/lib/mkspecs",
+                        "rm -rfv /app/pkgconfig",
+                        "rm -rfv /app/plugins/designer",
+                        "rm -rfv /app/sbin",
```
Comment (aleixpol) [inline io.qt.qtwebengine.BaseApp.json:101]: Won't this be removing stuff from the app?

**Comment by aleixpol** [comment]
Anyhow, LGTM, +1 overall

File: io.qt.qtwebengine.BaseApp.json (hunk 1 above)
Comment (eszlari) [reply io.qt.qtwebengine.BaseApp.json:101]: `sbin` contains:
* `lspci`
* `setpci`
* `update-pciids`

I didn't notice problems at runtime, when testing:

https://github.com/flathub/io.github.maurycyliebner.enve

Comment (aleixpol) [reply io.qt.qtwebengine.BaseApp.json:101]: Let's go with it and address it if it becomes a problem then.

**Pull request merged**